src/Raven.Studio/typescript/common/shell/clusterTopologyManager.ts

```diff
@@ -42,6 +43,7 @@ class clusterTopologyManager {
 
     private onTopologyUpdated(e: Raven.Server.NotificationCenter.Notifications.Server.ClusterTopologyChanged) {
         this.topology().updateWith(e);
+        licenseModel.fetchLicenseStatus();
```

Comment:
should we throttle this (or even debounce)? Please have a look at this: http://issues.hibernatingrhinos.com/issue/RavenDB-10008  I hope right now we don't get TopologyChanged too often.

Reply:
TopologyChanged for the cluster _should_ be rare